### ISSUE TITLE
Weekly Community Check-In #102 - Tech podcasts / newsletters recommendations 

### ISSUE THREAD

**Issue by cesswairimu** [publiclab/plots2#9614]
Hi everybody! 👏

We all at Public Lab 🎈 - learn, grow, work, brainstorm ideas, contribute together so why not share about our weekly goals and the awesome work we have done at Public Lab with each other, so we can support and collaborate with each other better. We have a Community Check-In each week, where every community member can share something about their work from the past week and about their current week's goal 🎯 . You are also welcome to share fun-fact 😄 , new ideas 💡 , your learning goals ☑️.

We believe in collaborative efforts to support our community. We are running a learning platform which helps a newcomer to become a leader of tomorrow. 💯

##  Tech podcasts / newsletters recommendations 

This week lets share tech newsletters or podcasts that we love and keeps us updated on what is happening with our favorite technologies.

## This week's focus Issues

These are issues that are of high priority and requests from our
large community that use the software. 

https://github.com/publiclab/plots2/labels/high-priority

We will very much appreciate any help on these issues. Any contributor who is looking for issues to work on please consider taking a look at these.
We will add a thank you message here in next week's check-in if you contribute to any of these. Thanks in advance 😄

### Thank you note to focus issues contributors

Many thanks to contributors of the past two weeks
 🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇
@jywarren, @Manasa2850, @keshav234156, @Hassanelashram, @ampaire, @TildaDares, @ramarahul, @rkpattnaik780, @j-saurabh, @tanishghosh, @sobekcore, @icarito, @VarunAroraCode, @AlanLake, @DR-FREKE, @dependabot[bot], @Komachian, @bunnyrockz, @gaurav2699, @roli25, @jcads, @RuthNjeri, @aliciapaz, @OlaoluwaM, @eightballocto
 🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇🥇

## Next Check-in
If you would like to open the next check-in..Leave a comment below. We are happy to help if its your first time

**Thanks everyone for making Public Lab awesome** :heart:  
Have a great week ahead all :balloon:

**Comment by jywarren** [comment]
Hi all! I really enjoy the Code Newbie podcast for the folks they interview and the themes they cover, and especially how they approach them. Strongly recommended! For example "What it’s like to break into tech as a mother with Arit Amana" or "Why ethics and contributor behavior matters in open source with Coraline Ada Ehmke":

https://www.codenewbie.org/podcast

I also quite enjoy some episodes of the createCanvas podcast from the Processing Foundation!

https://soundcloud.com/processingfoundation

Finally this is a little broader but I have really enjoyed (and even was lucky enough to be interviewed by!) the Abolition Science Radio podcast: 

https://www.abolitionscience.org/

**Comment by grvsachdeva** [comment]
Hi folks, 

Love all the suggestions posted above 😄 

I mostly read about system design, business, and start-ups, etc. Here are some suggestions:

1. https://kislayverma.com/blog-home/ - Covers system design topics and engineering practices, and also book recommendations, highlights, etc

2. https://ajuniorvc.com/ - Write about startup journeys, ideas, concepts around business, product, etc

3. Enjoy tweets of https://twitter.com/shreyas around Product, and sometimes watch tech conf videos at https://www.youtube.com/channel/UCkQX1tChV7Z7l1LFF4L9j_g

Happy weekend to all!

**Comment by cesswairimu** [comment]
Hi everyone, thanks all for sharing, loving all your recommendation.  

I love command line heroes and code newbie too :heart: 
here are other of my favorites 
- https://www.greaterthancode.com/
- https://www.ladybug.dev/
- https://devchat.tv/show/ruby-rogues/
and https://rubyweekly.com/ for newsletters

Would anyone like to open next week's check-in?  thanks in advance :)

**Comment by waridrox** [comment]
> Would anyone like to open next week's check-in? thanks in advance :)

@cesswairimu, would be my pleasure! 🙋🏻‍♂️😊

**Comment by cesswairimu** [comment]
Awesome thanks @waridrox 🎉

**Comment by RuthNjeri** [comment]
Hi Everyone 👋🏾 , this is an amazing checking @cesswairimu 🎉 

Thanks for sharing such awesome newsletters and podcasts. I have bookmarked all of them.
I used to listen to code newbie a lot on my way to work when I got my first programming job... nostalgia!

My recommendation is not exactly a tech-centric podcast, actually, it really isn't, but I think it will be super useful to anyone interested in/is in a leadership role, or wants to learn how to have better interactions with workmates, friends, and family. 

[Unlocking us and Dare To Lead](https://brenebrown.com/podcasts/) by Brene Brown. 

Those podcasts have helped me have more courage in speaking up for myself, sharing my thoughts, and bravely asking questions when I have no idea what is happening or confused. 

Another one is [how I built this](https://www.npr.org/podcasts/510313/how-i-built-this) by Guy Raz. I mostly listen to it when I am washing dishes, it is about how successful entrepreneurs built their businesses.

I also listen to some local podcasts just to keep up with what is happening, if you are Kenyan you might be interested in a special segment by the Kenyan Wallstreet called [we are curious](https://podcasts.apple.com/ke/podcast/kenyan-wallstreet/id1438822031). They talk about tech, finance..mostly finance and the intersection between the two.
